production_asset_url.json

```diff
@@ -1 +1 @@
-{"url":"https://d1x6jdqbvd5dr.cloudfront.net/e56291feb02a9a5a7a3c"}
\ No newline at end of file
+{"url":"https://d1x6jdqbvd5dr.cloudfront.net/ddd3db6372e540abba68"}
```

Comment:
Need to rebase from development and deploy assets again. Sorry!

Reply:
no worries, will do that. Thanks.